### ISSUE TITLE
Disable scanning of external (USB) drives

### ISSUE THREAD

**Issue by regs01** [nextcloud/android#12090]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

Connect external drive.

### Expected behaviour

Nextcloud should ask for each external drive with notification each time they connected.
As well as should be an option to disable it completely notification completely.

### Actual behaviour

Nextcloud starts scanning it instantly draining the battery to zero and spamming with thousands of notifications.

### Android version

10.0

### Device brand and model

Huawei

### Stock or custom OS?

Stock

### Nextcloud android app version

3.26

### Nextcloud server version

27

### Using a reverse proxy?

I don't know

**Comment by joshtrichards** [comment]
> Nextcloud starts scanning it instantly draining the battery to zero and spamming with thousands of notifications about new folders with media, beeping on each. Beep beep beep, for half a hour if there are thousands of them.
> 
> Nextcloud should ask for each external drive with notification each time they connected.

I don't believe the scanning itself is Nextcloud, but Android's Media Scanner. The notifications however...

> As well as should be an option to disable notification completely.

If you expand the first notification there is a button to the left that says *Disable* which will do precisely that. It disables notifications about any new found media folders from the media scan.

**Comment by regs01** [comment]
> I don't believe the scanning itself is Nextcloud, but Android's Media Scanner. The notifications however...

If Nextcloud dropping notifications, then i guess it does some kind of scanning as well.

> If you expand the first notification there is a button to the left that says _Disable_ which will do precisely that. It disables notifications about any new found media folders from the media scan.

Wouldn't that disable Nextcloud notifications entirely? But still, it would be better to disable it for USB devices with just one notification, whatever to scan for new folders or not.
